tyk-data-plane/values.yaml

```diff
@@ -0,0 +1,382 @@
+# Default values for tyk-data-plane.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+global:
+  isDataplane: true
+  components:
+    # Determines whether or not the pump component should be installed.
+    pump: false
+  rpc:
+    # connection string used to connect to an MDCB deployment
+    connectionString: ""
+    # orgID of your dashboard user
+    key: ""
+    # API key of your dashboard user
+    apiKey: ""
+    # needed in case you want to have multiple data-planes connected to the same redis instance
+    groupID: ""
+    # enable/disable ssl
+    useSSL: true
+    sslInsecureSkipVerify: true
```

Comment:
It would be helpful to have a brief description of what this field entails. 

Reply:
Added.

Reply:
i guess this file is not updated. i cannot see the comment.